tools/cassandra_stress_ci.py

```diff
@@ -0,0 +1,59 @@
+import os
+from collections import namedtuple
+
+import pytest
+from util.scylla_docker import ScyllaDockerNode
+from util.cassandra_stress import CassandraStress, CqlStressCassandraStress, CSCliRuntimeArguments
+from test_cs_write_and_validate import run as run_write_and_validate
+
+# GH actions log grouping utilities
+
+
+def print_in_ci(s):
+    if "CI" in os.environ:
+        print(s)
+
+
+def start_ci_group(group_name):
+    print_in_ci(f"\n::group::{group_name}")
+
+
+def end_ci_group():
+    print_in_ci("\n::endgroup::")
+
+
+# Utils for test cases
+
+@pytest.fixture
+def default_runtime_args():
+    return CSCliRuntimeArguments(workload_size="100", concurrency="1")
+
+
+DEFAULT_SCYLLA_URI = "127.0.0.1:9042"
+
+
+@pytest.fixture
+def scylla_docker_node():
+    scylla_uri = os.getenv("SCYLLA_URI", DEFAULT_SCYLLA_URI).split(':', 1)
+    return ScyllaDockerNode(ip=scylla_uri[0], port=scylla_uri[1])
+
+
+@pytest.fixture
+def cassandra_stress():
+    return CassandraStress()
+
+
+@pytest.fixture
+def cql_stress():
+    return CqlStressCassandraStress()
+
+
+# Test cases
+
+
+def test_write_and_validate(default_runtime_args, scylla_docker_node,
+                            cassandra_stress, cql_stress):
+    start_ci_group("C-S test 1")
+    run_write_and_validate(runtime_args=default_runtime_args, node=scylla_docker_node,
+                           cs=cassandra_stress, cql_stress=cql_stress)
+    end_ci_group()
```

Comment:
I think we should rather remove the `start_ci_group` and `end_ci_group` methods altogether. It will give weird results, e.g. pytest prints all logs from the failing tests at the end - so github would see them more than once.

Besides, side note for the future: if you want to make sure that something happens even if the test fails (i.e. `end_ci_group`), then it should happen in a teardown method (correspondingly, `start_ci_group` would be printed in setUp).